spec/spec_helper.rb

```diff
@@ -18,16 +18,17 @@ def create_test(client, service, method_name, parent_object)
   request_body = JSON.parse(json_from_file("mocks/requests/#{service}/#{method_name}.json"))
   response_body = json_from_file("mocks/responses/#{service}/#{method_name}.json")
 
-
   # client-generated headers
   headers = {
     "Content-Type".to_sym => "application/json",
   }
 
   # authentication headers
-  if not client.api_key.nil?
+  if !client.api_key.nil?
```

Comment:
Yah, that's find. I feel a little allergic to the "not" syntax since it has a different precedence than "!" and that can get confusing when they're mixed. I've historically just shied away from them. In any event, I'll change it back :(